frontend/src/pages/RoomsPages/TopicsList.tsx

```diff
@@ -9,6 +9,7 @@ type TopicsListProps = {
 type Topic = {
   id: number;
   topic: string;
+  rooms: string;
```

Comment:
are you sure that `rooms` are a type of string? I rather think that it's string[] but I am not sure whats under the hood